src/components/auth/LoginContainer.stories.tsx

```diff
@@ -2,26 +2,26 @@ import React from 'react'
 import { Meta, Story } from '@storybook/react'
 import { Provider } from 'react-redux'
 import { createStore } from 'redux'
-import { LoginContainer, LoginContainerProps, LoginStage } from './LoginContainer'
+import { LoginContainer, LoginContainerProps } from './LoginContainer'
 
-export default {
-  title: 'Explorer/Login',
-  args: {
-    stage: LoginStage.LOADING,
-    subStage: ''
-  } as LoginContainerProps,
-  component: LoginContainer,
-  argTypes: {
-    onLogin: { action: 'signing in...' },
-    onTermsChange: { action: 'terms changed...' }
-  }
-} as Meta
+// export default {
```

Comment:
Is there any reason why we keep commented code here?

Reply:
because it was failing silently, with the fixed storybook it really fails https://github.com/decentraland/explorer-website/runs/5758397649?check_suite_focus=true#step:7:30